test/cpp/naming/resolver_component_test.cc

```diff
@@ -279,8 +279,8 @@ void OpenAndCloseSocketsStressLoop(int dummy_port, gpr_event* done_ev) {
     }
     std::vector<int> sockets;
     for (size_t i = 0; i < 50; i++) {
-      SOCKET s = WSASocket(AF_INET6, SOCK_STREAM, IPPROTO_TCP, nullptr, 0,
-                           WSA_FLAG_OVERLAPPED);
+      SOCKET s = grpc_create_wsa_socket(AF_INET6, SOCK_STREAM, IPPROTO_TCP,
+                                        nullptr, 0, grpc_wsa_socket_flags);
```

Comment:
nit: let's keep the test simple and as is.... this test doesn't have a use for `WSA_FLAG_NO_HANDLE_INHERIT`